tests/error/inout_errors/unused_after_call.err

```diff
@@ -0,0 +1,7 @@
+Guppy compilation failed. Error in file $FILE:16
+
+14:    @guppy(module)
+15:    def test(q: qubit) -> None:
+16:       foo(q)
+              ^
+GuppyError: Variable `q` with linear type `qubit` is not used on all control-flow paths
```

Comment:
Same problem here. Any ideas what a good error message would be?

Reply:
Can we detect when `q` doesn't get used in **any** control-flow paths?
Then we can assume the user though this is a consuming operation, and we can emit something like
```
GuppyError: Variable `q` with linear type `qubit` is not consumed by this call
```